site/content/user-guide/jmeter_load_tests.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "JMeter Load Tests"
+weight = 300
```

Comment:
This just means that the current weights in Developer Guide and User Guide are messed up and need to be fixed.